src/uct/tcp/tcp.h

```diff
@@ -280,13 +280,35 @@ typedef struct uct_tcp_ep_zcopy_tx {
 } uct_tcp_ep_zcopy_tx_t;
 
 
+/**
+ * TCP iface address flags
+ */
+typedef enum uct_tcp_iface_addr_flags {
+    UCT_TCP_IFACE_ADDR_FLAG_LOCAL = UCS_BIT(0) /* Local address extension is
```

Comment:
They have different meanings and included in different fields.
Rename this flag to make it more clear, since it indicates whether we have `ext_addr` valid or not.